### ISSUE TITLE
Rebuilding docs for old crate

### ISSUE THREAD

**Issue by sww1235** [rust-lang/docs.rs#2084]
Hello:

Is it possible to rebuild the docs for older crates, that don't have the most recent theming / options available?

I came across the following crate: https://docs.rs/afm/latest/afm/ which was last updated in 2018, and so has not been rebuilt by docs.rs since. It is quite jarring to be pushed to a light mode page when my defaults are the ayu theme. 

it looks like the menu bar automatically updates, but the content does not (presumably because it is static content).

Not a huge issue, but I figured it might be worth bringing up.

A possible fix might be automatically queuing rebuild jobs for docs pages older than some metric (most recent HTML change, 2 years, most recent rustdoc release, etc) for timeslots when there are no other jobs running. Prioritize by oldest repositories and you will hit some of the low hanging fruit. This probably would take a while but would be a set and forget type of solution.

Thanks in advance.

**Comment by syphar** [comment]
This is a duplicate of #464 , and currently blocked on having more build capacity on our servers (which is WIP with our infra team)
